### PR TITLE
Create 20190609-白竹鼠玩沙子

### PR DESCRIPTION
**Pull request by immoonancient**
#67

### CODE REVIEW

**Comment by immoonancient** [comment]
Max could you review this? Thanks!

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+A walk on this pile of sand
+
+小狗帮我看着它 不要让它跑了
+Watch them for me, puppy! Don't let them run away
+
+跑到山上去 我们就捉不到它了
+If they run into the mountain, we can't catch them any more
+
+那么陡的 那么陡的沙埔它都可以爬得上去的哦
+They can even climb such a steep sand pile
```
Comment (vincenthychow) [inline subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]: They can even climb such a steep...steep sand pile.

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+这个竹鼠应该会打洞的 现在都不打洞了
+Bamboo rats are supposed to know how to dig. But they don't dig any more
+
+被我关太久了
+Maybe I caged them for too long
+
+失去了 失去了那个野性
+They have lost their... their wild nature
```
Comment (vincenthychow) [inline subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]: They have lost...lost their....eh...wild nature

劉蘇良很多"這個"，"那個"，我通常都會用eh...或其他方法表達出來

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+Look at that one. Running so fast
+
+还一扭一扭的屁股
+And the ass moves back and forth
+
+呃 把它领回去
+Eh... Let's take it back
+
+把它回去 这个又跑啦
+This one is back. Now that one is running
```
Comment (vincenthychow) [inline subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]: Now that one is running away

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi (hunk 1 above)
Comment (vincenthychow) [inline subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]: They can even climb such a steep...steep sand pile.

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi (hunk 2 above)
Comment (vincenthychow) [inline subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]: They have lost...lost their....eh...wild nature

劉蘇良很多"這個"，"那個"，如果並非用作指代詞的話，我通常都會用eh...或其他方法表達出來

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi (hunk 3 above)
Comment (vincenthychow) [inline subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]: Now that one is running away

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi (hunk 1 above)
Comment (immoonancient) [reply subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]:
```suggestion
They can even climb such a steep... steep sand pile
```
Reformatted as a suggestion comment to ease revision...

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi (hunk 2 above)
Comment (immoonancient) [reply subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]:
```suggestion
They have lost... lost their.... eh... wild nature
```

File: subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi (hunk 3 above)
Comment (immoonancient) [reply subtitles/hua-nong-brothers/20190609-bai-zhu-shu-wan-sha-zi]:
```suggestion
This one is back. Now that one is running away
```

**Pull request merged**